src/app/Http/Controllers/Auth/ForgotPasswordController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\ResetLinkRequest;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Support\Facades\Password;
+
+class ForgotPasswordController extends Controller
+{
+    // Show the forgot password form
+    public function showLinkRequestForm(): View
+    {
+        return view('auth.passwords.email');
+    }
+
+    // Send a password reset link to the user
```

Comment:
Use docbloc but don't include the @param or @return on it because you already assigned the types of your parameter and return on your method. Kind a redundant right??


https://docs.phpdoc.org/guide/getting-started/what-is-a-docblock.html